modules/rpc/kull_m_rpc_ms-efsr_c.c

```diff
@@ -14,7 +14,7 @@ extern const UCHAR ms2Defsr__MIDL_ProcFormatString[271];
 #endif
 extern const UCHAR ms2Defsr__MIDL_TypeFormatString[55];
 
-const RPC_CLIENT_INTERFACE efsrpc___RpcClientInterface = {sizeof(RPC_CLIENT_INTERFACE), {{0xc681d488, 0xd850, 0x11d0, {0x8c, 0x52, 0x00, 0xc0, 0x4f, 0xd9, 0x0f, 0x7e}}, {1, 0}}, NDR_TSI_20, 0, 0, NULL, 0, 0, 0x00000001};
+const RPC_CLIENT_INTERFACE efsrpc___RpcClientInterface = { sizeof(RPC_CLIENT_INTERFACE), {{0xdf1941c5, 0xfe89, 0x4e79, {0xbf, 0x10, 0x46, 0x36, 0x57, 0xac, 0xf4, 0x4d}}, {1, 0}}, NDR_TSI_20, 0, 0, NULL, 0, 0, 0x00000001 };

```

Comment:
Was this using the lsarpc UUID before and thus incorrect? Just trying to figure out why the constant needs to be updated.

Reply:
yes. previous UUID was associated to `\pipe\lsarpc` and this updated version is related to `\pipe\efsrpc`. Microsoft link [here](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-efsr/1baaad2f-7a84-4238-b113-f32827a39cd2), as I mentioned on [issue](https://github.com/rapid7/metasploit-payloads/issues/608).

Reply:
Marking resolved, this matches what was linked in @NtAlexio2's comment above and looks correct from these links.